src/libs/NumberFormatUtils.ts

```diff
@@ -1,12 +1,19 @@
+import moize from 'moize';
 import type {ValueOf} from 'type-fest';
 import type CONST from '@src/CONST';
 
+const numberFormatter = moize(Intl.NumberFormat, {
+    isDeepEqual: true,
+    maxSize: 10,
```

Comment:
question, how do we determine the maxSize to be 10?

Reply:
It's arbitrary number based on the testing, where we did not tend to go beyond such cache size. We can adjust that if there is better reasoning 👍

Reply:
NAB - just to confirm how this works - with maxSize 10, is that only ten sets of in this case `locale`, `options`, and `number`? It seems like we'd need a lot more than 10... but is the performance issue right now that this gets called multiple times in related requests with the same inputs?